### PR TITLE
Login message send

### PR DESCRIPTION
**Pull request by stevejarvis**
Can start both the server and client on the command line and "login" to the server.

Everyone able to reproduce that? You should all be able to. There's a lot of "TODOs" marked here, but I wanted to get it up so we have a foundation to get messages flowing.

### CODE REVIEW

File: server/src/main/java/org/server/SupServer.java [hunk 1]
```diff
@@ -2,90 +2,114 @@
...
+            }
         }
-      }
-      catch(Exception ex) {
-        ex.printStackTrace();
-      }
-    }
-  }
-  
-  public void start () {
```
Comment (stevejarvis) [inline server/src/main/java/org/server/SupServer.java:54]: these didn't really change, i just replaced tabs with 4 spaces to match our coding standard. probably should have done that in a different commit...

**Comment by izoubi** [comment]
The code is great, easy to read and understand and most importantly does the job.  
Approve to merge.
